subsys/bluetooth/audio/endpoint.h

```diff
@@ -53,6 +53,10 @@ struct bt_audio_ep {
 	struct bt_audio_broadcast_sink *broadcast_sink;
 };
 
+struct bt_audio_ep_info {
+	struct bt_audio_ep *ep;
+};
```

Comment:
The struct should contain the value that you want from the `ep`. Since `ep` is an opaque struct, just getting the pointer won't do much good. 

The struct should rather have a field
```c
uint8_t ase_id
```

and assigned like 
```c
ase_id = ep->status.id;
```

We could also add the state (`ep->status.state`) to it, as that has been requested by https://github.com/zephyrproject-rtos/zephyr/issues/46238